common/build.gradle

```diff
@@ -14,11 +14,11 @@ dependencies {
     // Compile-time dependencies that should NOT be part of the
     // shadow jar and are provided in the lib folder of Flink
     // --------------------------------------------------------------
-    shadow "org.apache.flink:flink-runtime_${scalaBinaryVersion}:${flinkVersion}"
+    shadow "org.apache.flink:flink-runtime:${flinkVersion}"
```

Comment:
nit: actually, this dependency can also be removed since it is a transitive dependency from the ones defined in the main `build.gradle` file at the root project